Packs/GitGuardian/Integrations/GitGuardianEventCollector/GitGuardianEventCollector.py

```diff
@@ -0,0 +1,383 @@
+import demistomock as demisto
+from CommonServerPython import *
+import urllib3
+from typing import Any
+
+# Disable insecure warnings
+urllib3.disable_warnings()
+
+""" CONSTANTS """
+
+DATE_FORMAT = "%Y-%m-%dT%H:%M:%S.%fZ"
+VENDOR = "gitguardian"
+PRODUCT = "enterprise"
+DEFAULT_PAGE_SIZE = 1000
+EVENT_TYPE_TO_TIME_MAPPING = {
+    "audit_log": "gg_created_at",
+    "incident": "last_occurrence_date",
+}
+EVENT_TYPE_TO_ENDPOINT = {
+    "audit_log": "/audit_logs",
+    "incident": "/secrets",
+}
+
+""" CLIENT CLASS """
+
+
+class Client(BaseClient):
+    """Client class to interact with the service API
+
+    This Client implements API calls, and does not contain any Demisto logic.
+    Should only do requests and return data.
+    It inherits from BaseClient defined in CommonServer Python.
+    Most calls use _http_request() that handles proxy, SSL verification, etc.
+    For this HelloWorld implementation, no special attributes defined
+    """
+
+    def search_events(
```

Comment:
I'd merge this function with `fetch_events()`

Reply:
I prefer to keep them as separate functions